files/help/ares-log.help

```diff
@@ -0,0 +1,97 @@
+{
+    "display" : ["name", "synopsis", "options", "description", "examples"],
+    "notab":["examples"],
+    "name" : "ares-log - Show and save logs of webOS's apps and services",
+    "synopsis" : {
+        "default" : [
+           "ares-log [OPTION...]",
+            "ares-log [OPTION...] -f, --follow",
+            "ares-log [OPTION...] -r, --reverse",
+            "ares-log [OPTION...] -n, --lines <LINE>",
+            "",
+            "ares-log [OPTION...] -b, --boot",
+            "ares-log [OPTION...] -k, --dmesg",
+            "ares-log [OPTION...] -p, --priority <PRIORITY>",
+            "ares-log [OPTION...] -pid, --pid <PID>",
+            "ares-log [OPTION...] -S, --since <DATE>",
+            "ares-log [OPTION...] -U, --until <DATE>",
+            "",
+            "ares-log [OPTION...] -ul, --unit-list",
+            "ares-log [OPTION...] -u, --unit <UNIT>",
+            "",
+            "ares-log [OPTION...] -fl, --file-list",
+            "ares-log [OPTION...] -file, --file <JOURNAL_FILE>"
+        ]
+    },
+    "description" : [
+        "This command shows and saves log of webOS",
```

Comment:
It would be better to add description about "current ares-log is working with journalctl "